Mathlib/MeasureTheory/MeasurableSpace/CountablyGenerated.lean

```diff
@@ -166,35 +166,73 @@ theorem SeparatesPoints.mono {m m' : MeasurableSpace α} [hsep : @SeparatesPoint
     @SeparatesPoints _ m' := @SeparatesPoints.mk _ m' fun _ _ hxy ↦
     @SeparatesPoints.separates _ m hsep _ _ fun _ hs ↦ hxy _ (h _ hs)
 
+/-- We say that a measurable space is countably separated if there is a
+countable sequence of measurable sets separating points. -/
+class CountablySeparated (α : Type*) [MeasurableSpace α] : Prop where
+  countably_separated : HasCountableSeparatingOn α MeasurableSet univ
+
+instance countablySeparated_of_hasCountableSeparatingOn [MeasurableSpace α]
+    [h : HasCountableSeparatingOn α MeasurableSet univ] : CountablySeparated α := ⟨h⟩
+
+instance hasCountableSeparatingOn_of_countablySeparated [MeasurableSpace α]
+    [h : CountablySeparated α] : HasCountableSeparatingOn α MeasurableSet univ :=
+  h.countably_separated
+
+theorem countablySeparated_def [MeasurableSpace α] :
+    CountablySeparated α ↔ HasCountableSeparatingOn α MeasurableSet univ :=
+  ⟨fun h => h.countably_separated, fun h => ⟨h⟩⟩
```

Comment:
Could you replace `=>` by `↦` for functions everywhere in you code?
```suggestion
  ⟨fun h ↦ h.countably_separated, fun h ↦ ⟨h⟩⟩
```
I checked the style guide to be sure it's not just my personal preference: https://leanprover-community.github.io/contribute/style.html contains "The Lean default for this is `fun x => x * x`, but the `↦` arrow (inserted with \mapsto) is also valid. In mathlib the pretty printer displays `↦`, and we slightly prefer this in the source as well."

There are many `=>` still in mathlib but they are mostly due to the automatic Lean3 to Lean4 translation done by mathport.

Reply:
I think I got them all, thanks.